### PR TITLE
enhanced and modified footer issue #131

### PR DESCRIPTION
**Pull request by coding-geek21**
## Related Issue or bug
  - Enhancement of footer part in common.html and home.html
  
Fixes: #131 

#### Describe the changes you've made

1) Modified footer in common.html and home.html
2) Added quick links with icons 
3) Added social media icons 
4) Added copyrights to the website

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Tested the responsiveness of the footer part in google chrome

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 **original screenshot**  | <b>updated screenshot </b> |
 
 
 
image

### CODE REVIEW

File: main_app/templates/main_app/common.html [hunk 1]
```diff
@@ -21,6 +21,9 @@
       href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"
```
Comment (sakshikhachane) [inline main_app/templates/main_app/common.html:21]: The `login` page is getting distorted. Please fix it.
image

File: main_app/templates/main_app/common.html [hunk 2]
```diff
@@ -354,40 +405,39 @@ <h4 class="new">&nbsp;&nbsp; More</h4>
...
+           </ul>
+       </div>
+        <div class="col-xs-12 col-sm-4 col-md-4" style="padding-left:20px;">
+          <h3 style="border-left: 5px solid white;padding:8px;margin-left:10px;">Contact us</h3>
+          <br/><br/>
+          <a href="#"><i style="background: #55ACEE;" class="bi bi-twitter ic"></i></a>
+          <a href="#"><i style="background: #3B5998;" class="bi bi-facebook ic"></i></a>
+          <a href="#"><i style="background: #007bb5;" class="bi bi-linkedin ic"></i></a>
+          <a href="#"><i style="background: salmon;" class="bi bi-envelope-fill ic"></i></a>
+          <a href="#"><i style="background:purple;" class="bi bi bi-github ic"></i></a>
```
Comment (sakshikhachane) [inline main_app/templates/main_app/common.html:434]: Change the background color of the GitHub icon to black and for now, add the GitHub link of the `rescue` project to all the icons.

**Review by sakshikhachane — CHANGES_REQUESTED** [review]
@coding-geek21 The footer is looking good. Just do some minor changes.

File: main_app/templates/main_app/common.html (hunk 2 above)
Comment (coding-geek21) [reply main_app/templates/main_app/common.html:434]: Okay @sakshikhachane  i will make all the changes

File: main_app/templates/main_app/login.html [hunk 3]
```diff
@@ -3,31 +3,33 @@
 {% load static %}
```
Comment (sakshikhachane) [inline main_app/templates/main_app/login.html:3]: @coding-geek21 On the` login` page, the form and footer are now not overlapping but the position of both footer and form is distorted on the small screen. Can you please fix it?

**Review by sakshikhachane — CHANGES_REQUESTED** [review]

**Comment by coding-geek21** [comment]
Hi @sakshikhachane  There is already distortion in login.html page before modification of footer (this issue)
image

image

I think there is already a bug in login.html page and i worked only on footer part i don't know how to fix this issue, but my footer added in common.html  works good in every pages except this one so i think there is no problem in footer part and there is problem only in login.html page , can you give an idea on how to fix this , or i guess someone is working on login page issue maybe they can fix this

**Review by sakshikhachane — CHANGES_REQUESTED** [review]
Hey @coding-geek21,
There is space left after the footer.
image

**Comment by coding-geek21** [comment]
> Hey @coding-geek21,
> There is space left after the footer.
> image


image
@sakshikhachane  Hi can you please check whether it is okay now?

**Comment by sakshikhachane** [comment]
@coding-geek21 Yes it is looking better just increase the top margin of the form. Then it is ready to be merged.

**Comment by coding-geek21** [comment]
Hey @sakshikhachane  I fixed the white space after the footer in login page


image

**Comment by coding-geek21** [comment]
Hi @sakshikhachane  I have made changes and added margin please review the changes

**Review by sakshikhachane — APPROVED** [review]

**Comment by sakshikhachane** [comment]
@coding-geek21 Please resolve the conflicts.

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
Few Changes:
1. Change the colour to navbar colour.
2. Social media icons are stretching, fix it.
3. Under quick links sections, remove home and emergency and add Developers page and remove the developer link from rescue column.

**Comment by coding-geek21** [comment]
> Few Changes:
> 
> 1. Change the colour to navbar colour.
> 2. Social media icons are stretching, fix it.
> 3. Under quick links sections, remove home and emergency and add Developers page and remove the developer link from rescue column.

image


Hello @swapnilsparsh please review the changes

**Comment by swapnilsparsh** [comment]
@coding-geek21  
Social media icons are stretching, fix it and remove the copyright line.

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
Few changes:
1. Remove all the social media icons and add a blank link there redirecting to contact form that is being under development.
2. Remove the heading Quick links and vertical lines

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
Footer

Move the marked headings in center and keep the contact us heading there only.

**Comment by coding-geek21** [comment]
image

@swapnilsparsh I did the requested changes please review it

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
f

While checking the footer for small screens , on some pages there is bug in footer.

**Comment by coding-geek21** [comment]
> f
> 
> While checking the footer for small screens , on some pages there is bug in footer.

@swapnilsparsh I will fix it

**Comment by sakshikhachane** [comment]
@coding-geek21 Any updates?

**Comment by coding-geek21** [comment]
> @coding-geek21 Any updates?

I will make commits by tomorrow

**Comment by coding-geek21** [comment]
@swapnilsparsh  fixed the whitespace after footer please review it

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
Resolve the conflicts

**Comment by swapnilsparsh** [comment]
Footer

Fix it

**Comment by coding-geek21** [comment]
@swapnilsparsh  fixed the issues please review it

**Comment by coding-geek21** [comment]
@swapnilsparsh  fixed the issues please review it

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
To top button not working, fix it

**Comment by coding-geek21** [comment]
> To top button not working, fix it

@swapnilsparsh  fixed it please review it

**Pull request merged**